packages/node/src/utils.ts

```diff
@@ -73,9 +75,69 @@ export enum EdgeRuntimes {
   ExperimentalEdge = 'experimental-edge',
 }
 
+export enum NodejRuntime {
+  // @ts-ignore typescript does not allow undefined as a value
+  Legacy = undefined,
+  WinterCG = 'nodejs-experimental', // TODO remove -experimental once ready
+}
+
+export const ALLOWED_RUNTIMES = [
+  NodejRuntime.WinterCG as string,
```

Comment:
Now you can change this to `Object.values(NodejsRuntimes)` right?

Reply:
We actually can not, because `NodejsRuntime` contains `Legacy`, which is `undefined`. It will look quite bad in error message, where we display items of thie array.